riscv/processor.cc

```diff
@@ -453,7 +453,8 @@ void state_t::reset(processor_t* const proc, reg_t max_isa)
                               (proc->extension_enabled(EXT_SVADU) ? MENVCFG_ADUE: 0) |
                               (proc->extension_enabled(EXT_SVPBMT) ? MENVCFG_PBMTE : 0) |
                               (proc->extension_enabled(EXT_SSTC) ? MENVCFG_STCE : 0);
```

Comment:
`SSE` should be added to `menvcfg_mask`, `senvcfg_mask`, and `henvcfg_mask` if ZICFISS extension is enabled. This is missing.